docs/sql/ddl.txt

```diff
@@ -812,3 +812,29 @@ node startup defined configuration file value or to its default value::
     +--------------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+
+Show Create Table
+=================
+
+To print a user-created table the ``SHOW CREATE TABLE`` statement can be used. This statement displays the DDL-schema of an existing table in the cluster. The following statement creates a table with ident ``my_table`` and defines it's schema. ::
+
+    cr> create table my_table (
```

Comment:
but make it a bit more complex (column constraint, routing column, table settings)